dev/bots/analyze.dart

```diff
@@ -1040,6 +1043,38 @@ Future<void> verifyNoTrailingSpaces(String workingDirectory, { int minimumMatche
   }
 }
 
+final RegExp _flowControlStatementWithoutSpace = RegExp(r'(^|[ \t])(if|switch|for|do|while|catch)\(');
```

Comment:
You can probably use `\b` instead of `(^|[ \t])` ?

Reply:
if you do want to use `(^|[ \t])` then you'll need to set `multiLine: true` to the RegExp constructor.

Reply:
Ah, yes, \b works here - thanks.

Reply:
Actually I'll revert to `(^|[ \t])` as with `\b` the following async `catch()` method invocation in Java now matches the RegExp:

```
await serviceWorkerLoader.loadServiceWorker(serviceWorker).catch(e => {
```

I'll add `multiLine: true` though I guess it shouldn't be strictly needed as the input is split into lines already via `readAsLinesSync()`;